src/main/resources/liquibase/scripts/student.sql

```diff
@@ -0,0 +1,8 @@
+-- liquibase formatted sql
+
+-- changeset mavile:1
+CREATE INDEX student_name_index ON student (name)
```

Comment:
Скрипты нужно разделять между собой с помощью ;
```suggestion
CREATE INDEX student_name_index ON student (name);
```